be/src/formats/parquet/scalar_column_reader.h

```diff
@@ -145,6 +146,7 @@ class ScalarColumnReader final : public RawColumnReader {
     }
 
     void set_can_lazy_decode(bool can_lazy_decode) override {
+        _can_lazy_convert = can_lazy_decode;
```

Comment:
Why set `_can_lazy_convert` using `can_lazy_decode`?

Reply:
we can treat convert as kind of decode, we set_can_lazy_decode for columns that don't have predicates.